tests/Enable.test.ts

```diff
@@ -87,4 +91,42 @@ describe(`EnableCommand`, () => {
       await expect(sortedEntries).resolves.toEqual(expectedEntries.sort());
     });
   });
+
+  testNotWindows(`should overwrite existing files`, async () => {
```

Comment:
https://vitest.dev/api/#test-skipif
```suggestion
  test.skipIf(process.platform === `win32`)(`should overwrite existing files`, async () => {
```